src/EditorFeatures/Core/CodeDefinitionWindowLocation.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Editor
+{
+    internal struct CodeDefinitionWindowLocation
+        {
```

Comment:
Mea culpa - I had this as a nested type, and moving the lines with VsVim didn't fix the indenting. :blush: